app/models/champs/commune_champ.rb

```diff
@@ -59,15 +59,15 @@ def code_postal=(value)
   alias postal_code code_postal
 
   def name
-    if code?
```

Comment:
Ce cas la ne doit pas arriver. Je préfère avoir une erreur plutôt que faire l'autruche.

Reply:
C’est le dossier 7077139 et je soupçonne que le code du département n’est pas ﻿nil, mais la chaîne de caractères ﻿`undefined`, ce qui fait que ma moulinette n’a pas corrigé le problème.

Reply:
`Champs::CommuneChamp.where.not(external_id: nil).where("value_json->>'code_departement' = ?", 'undefined').count` il y a plus d'un millier de champ comme ça. Il va falloir corriger.

Reply:
Il y a aussi des communes avec le code 99 ce qui n'est pas possible. Je serai assez curieux de voir ce qu'il y a dedans.

Reply:
Je te rejoins à 100%, si tu penses que c'est non nominal et qu'on peut corriger les erreurs sur les données. Je ferme cette PR :)